scripts/shared.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path')
+const spawn = require('cross-spawn')
+const fs = require('fs')
+
+const PACKAGES = [
+  'contentful-ui-extensions-sdk',
+  '@contentful/ui-extensions-sdk',
+  '@contentful/app-sdk',
```

Comment:
We can also add `contentful-app-sdk`

Reply:
I assumed that the library was first published as `contentful-ui-extensions-sdk` and then `@contentful/ui-extensions-sdk` was added when npm scopes were introduced. I therefore now only "only" added the scoped version of `app-sdk`.

Reply:
It's more about package name parking to avoid copy cats/name squatting.

Reply:
Makes sense. Just added it.